dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -256,9 +256,14 @@ HRESULT CDefaultContextMenu::_DoCallback(UINT uMsg, WPARAM wParam, LPVOID lParam
     return E_FAIL;
 }
 
+
 void CDefaultContextMenu::AddStaticEntry(const HKEY hkeyClass, const WCHAR *szVerb)
 {
     PStaticShellEntry pEntry = m_pStaticEntries, pLastEntry = NULL;
+    WCHAR wszDefault[40];
+    DWORD pdwType, dwSize = _countof(wszDefault);
+    HKEY hShellKey = 0;
+    LRESULT lres = RegOpenKeyExW(hkeyClass, L"shell", 0, KEY_READ, &hShellKey);
```

Comment:
There is a `return` below in L267/272 where you leak this handle. Should move this call to right above the RegQueryValue